views/leaderboard.pug

```diff
@@ -15,11 +15,37 @@ block content
   .container
     .row
       .col.s12
-        .card.center
-          .card-content
-            h4.center Leaderboard
-          .card-action
-            .leaderboard-content
+        .row
+          ul.tabs
+            li.tab.col.s3
+              a.noLineLink.active#option-overall(href='#') Overall
```

Comment:
whats the href for?

Reply:
Woops, another thing from earlier on that I forogt to remove.